### PR TITLE
chore(packages): update dependencies

### PR DESCRIPTION
**Pull request by Johann-S**
Update dependencies

### CODE REVIEW

**Comment by XhmikosR** [comment]
Maybe something you updated drops node 4 support?

Also, I'm not sure they want package-lock.json here since they seem to be using yarn.

**Comment by Johann-S** [comment]
You're right about package-lock.json 👍
But it's more related to Travis because I merged a PR this afternoon and same error

**Comment by XhmikosR** [comment]
For example https://github.com/sindresorhus/load-grunt-tasks/blob/master/.travis.yml

**Comment by johnjbarton** [comment]
I sent a PR to drop node 4, then you can retry.

**Comment by XhmikosR** [comment]
@Johann-S: can you rebase and update again the packages? It seems Node.js >= 6 is required now, which reminds me we should add the engines property in package.json later.

**Comment by XhmikosR** [comment]
BTW nuke any local node_modules and del package-lock.json before doing `npm i` after updating the deps.

**Comment by XhmikosR** [comment]
NVM, just noticed you made the branch in this repo, so I made the changes myself :)

**Comment by XhmikosR** [comment]
All right, tests pass. We only need to solve the CLA thing @johnjbarton, then merge this, bump the minor version and release it.

File: package.json [hunk 1]
```diff
@@ -24,26 +24,26 @@
...
-    "load-grunt-tasks": "^3.2.0",
-    "mocha": "^5.0.4",
-    "qunit": "^2.5.1"
+    "karma-jasmine": "2.x",
+    "load-grunt-tasks": "^4.0.0",
+    "mocha": "^6.0.2",
+    "qunit": "^2.9.2"
   },
   "peerDependencies": {
     "qunit": "^2.0.0"
```
Comment (XhmikosR) [inline package.json]: Hmm, shouldn't we include `karma` in peerDeependencies?

Comment (Johann-S) [reply package.json]: seems a good idea to me 👍

**Comment by XhmikosR** [comment]
Does anybody know how we can proceed with the CLA?

**Comment by XhmikosR** [comment]
I guess we are stuck then. We could drop this PR and I can make one PR with only my patch which should pass.

**Comment by Johann-S** [comment]
Seems good to me 👍

**Pull request closed without merging**